types/global.d.ts

```diff
@@ -17,4 +17,5 @@ declare module '*.json' {
 
 declare interface Window {
   prettierPlugins: any
+  resetExternalLayout: () => void
```

Comment:
shouldn't we declare this method as optional one, given that this type is globally applied to all scenarios and not just screener tests?

Reply:
Good catch 👍